templates/actors/ffg-adversary-sheet.html

```diff
@@ -109,7 +109,7 @@ <h2><input name="name" type="text" value="{{actor.name}}" placeholder="{{localiz
         {{!-- Force Box --}} {{#if (or (eq actor.flags.config.enableForcePool undefined) (eq actor.flags.config.enableForcePool true) )}} {{> "systems/starwarsffg/templates/parts/shared/ffg-block.html" (object blocktype="split" title="SWFFG.ForcePool" fields=(array (object name="data.stats.forcePool.value" value=data.stats.forcePool.value type="Number" label="SWFFG.ForcePoolCommitted") (object name="data.stats.forcePool.max" value=data.stats.forcePool.max type="Number" label="SWFFG.ForcePoolAvailable") ))}} {{/if}}
       </div>
 
-      {{> "systems/starwarsffg/templates/parts/actor/ffg-talents.html"}} {{!-- Force Powers List --}} {{#if (or (eq actor.flags.config.enableForcePool undefined) (eq actor.flags.config.enableForcePool true) )}} {{> "systems/starwarsffg/templates/parts/actor/ffg-forcepowers.html"}} {{/if}} {{> "systems/starwarsffg/templates/parts/actor/ffg-signatureability.html"}}
```

Comment:
Looks like this just needs updating to use the properly scoped flags

`actor.flags.config.enableForcePool` should instead be `actor.flags.starwarsffg.config.enableForcePool`

Reply:
I think that just setting Force Powers to only display when a character has one (matching the Signature Ability behavior) makes sense. Either way works and I am not focused on Star Wars so it may make more sense one way or another based on the rules.

Reply:
Hm. If the character doesn't have a Force Pool, it doesn't really make sense for them to be showing force powers.

However, does that flag-check mean that disabling the Force Pool effectively hides any force powers that may already be assigned to the character? That doesn't seem particularly helpful, either.

Reply:
With that flag, disabling the force pool in the sheet options for that character would disable the force pool (committed/maximum section) and the force abilities.

Reply:
This is entirely opinion. but adding:
`{{#if (or (eq actor.flags.starwarsffg.config.enableForcePool undefined) (eq actor.flags.starwarsffg.config.enableForcePool true) )}} [...] {{/if}}`
Just so that force powers are turned off even if you have them, rather than only basing force powers on if you have them seems like just another failure point that doesn't accomplish a goal. Unless I'm misunderstanding something, which is entirely possible!

Reply:
I did see that adversaries had the wrong flag for the Force Box, so I updated that to `actor.flags.starwarsffg.config.enableForcePool`

Reply:
I would rather keep the config toggle for the force powers as it previously was.

If a character doesn't have a force pool, then they can't use a force power, so it makes sense to have their visibility linked.

Reply:
IMO the flag just adds an additional failure point without having much value. But I'm new here and will implement whatever's decided on. 😄

Reply:
Introduced a compromise that I think will work well.
1) Don't show force power information if the unit doesn't have any
2) Don't show force power information if the unit doesn't have the dice pool enabled
3) If pool is disabled but unit has force powers show a conflict resolver.
![image](https://user-images.githubusercontent.com/6833267/212628027-7f477464-b383-4bfa-947d-ffd1c983f48f.png)

There is a minor issue that I was wondering if anyone knew how to fix. 
If you hit "Enable Dice Pool" and open the Sheet Options then the checkbox still isn't checked. Closing and reopening the sheet resolves this, but I dug into the sheetOptions data and it's set to true there, and there window is created on the fly, so I don't see how it can be that it's not getting updated.

I also split the original fix into its own commit in case this is declined it can just be cherry picked out. xD